en/docs/event-gateway/publish-apis/subscription-based-apis.md

```
@@ -0,0 +1,75 @@
+
+4. Pick a Subscription plan and click **Subscribe**.
+
+5. You will receive a Subscription Token.
+
+## Invoke the API
+
+1. Receive the **cURL** to invoke the API using the Subscription Token by navigating to the **Documentation**.
+
+2. Invoke the API.
+
+    #### Sample Request
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)